Assets/MixedRealityToolkit/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
It would be good to get in sync on keywords, author information and unity versions. Please see my package.json [here](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/feature/virtUnityPack/Assets/package.json). 2018.4 stable is the minimum recommended version in the [Getting Started](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithTheMRTK.html) guide. In my file I used the same keywords currently used by the MRTK [Project](https://github.com/microsoft/MixedRealityToolkit-Unity) itself. I'm not saying any set of keywords is better (and in fact they probably don't really matter) but it would be good to have them in sync across PRs. Same with contact information. I wasn't aware that e-mail address is optional. We can probably just leave that off, but it would be good to set the URL.